.github/workflows/linting.yml

```diff
@@ -63,16 +63,34 @@ jobs:
   markdown-link-check:
     name: Markdown Links (modified files)
     runs-on: ubuntu-latest
+    env:
+      BASE_BRANCH: ${{ github.base_ref }}
+      FILE_EXTENSION: .md
     steps:
       - name: Check out the repository
         uses: actions/checkout@v2
-
+      - name: Extract markdown from CSV.yaml
+        run: |-
+          sudo snap install yq --channel=v4/stable && \
```

Comment:
If we just did this link extraction, creating a new (uncommitted) file, would enabling the GHA's built-in `check-modified-files-only` flag include it in the checks?

If so, it seems we could do that and avoid creating our own modified-f-MD-iles finding logic.

Reply:
@dfarrell07 unfortunately no, this was my first approach but this action compares between committed files

Reply:
Per discussion on the Automation Sync, an option is to commit the extracted links file locally in the GHA runner (`git commit -am "Extracted links in YAML for linting"`). It shouldn't have any side-effects, and would allow us to avoid creating modified-MD-files logic ourselves.